modules/mtc-deploying-a-vm.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+// * migration_toolkit_for_containers/mtc-migrating-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="mtc-deploying-a-vm_{context}"]
+= Deploying a virtual machine
+
+After installing and activating {VirtProductName} and Containerized Data Importer (CDI), create a namespace and deploy a virtual machine (VM).
+
+.Procedure
+
+* Deploy the YAML, which creates both a VM definition and a data volume containing the Fedora operating system. 
++
+In the following example, the namespace `mig-vm` is used and the following YAML is used to create a Fedora VM,  create and a datavolume containing the Fedora operating system:
+
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: rhel9-lime-damselfly-72
+  namespace: mig-vm # <1>
+  labels:
+    app: rhel9-lime-damselfly-72
+    kubevirt.io/dynamic-credentials-support: 'true'
+    vm.kubevirt.io/template: rhel9-server-small
+    vm.kubevirt.io/template.namespace: openshift
+    vm.kubevirt.io/template.revision: '1'
+    vm.kubevirt.io/template.version: v0.31.1
+spec:
+  dataVolumeTemplates:
+    - apiVersion: cdi.kubevirt.io/v1beta1
+      kind: DataVolume
+      metadata:
+        name: rhel9-lime-damselfly-72
+      spec:
+        sourceRef:
+          kind: DataSource
+          name: rhel9
+          namespace: openshift-virtualization-os-images
+        storage:
+          resources:
+            requests:
+              storage: 30Gi
+  running: false # <2>
```

Comment:
change to `running: true`
since we have notes in the following lines:
`<2> Use `running: true` to indicate that the VM should be started after creation.`